src/webots/nodes/WbBallJoint.cpp

```diff
@@ -256,7 +256,6 @@ void WbBallJoint::updatePositions(double position, double position2, double posi
 }
 
 void WbBallJoint::updatePosition(double position) {
```

Comment:
It seems that there is no virtual base method for this function, if so then the override attribute could be removed.
Then, do we really need it for BallJoint and Hinge2Joint) or could it be simply deleted?

Reply:
It isn't really needed by the BallJoint and Hinge2 classes but they do serve a purpose in the base ones. They could be removed for the former two but it might be risky as well, all it takes is missing a "s" and inadvertently we call the parent method and might not be evident to debug. Wouldn't it be better instead to leave them and put an `assert(false)`?